### PR TITLE
Support C++

### LINKED ISSUE #104: Support c++
We should support c++ as language for gaia pipelines.

### PR DESCRIPTION
**Pull request by michelvocks**
Fixes #104 

**DONE**:
* First version of C++ Gaia SDK: https://github.com/gaia-pipeline/cppsdk
* C++ Pipeline can be executed via Gaia.
* Implement build functionaility for C++ pipelines.
* Tests
* Documentation: https://github.com/gaia-pipeline/docs/pull/4

### CODE REVIEW

**Comment by Skarlso** [comment]
Nice! :D I'll review it ASAP. :)

**Comment by michelvocks** [comment]
> Nice! :D I'll review it ASAP. :)

Thanks! 🤗 It's still WIP but I hope I can finish it by end of this week.

**Comment by Skarlso** [comment]
Dude that's a massive coverage drop. Are you okay with that?

**Comment by michelvocks** [comment]
> Dude that's a massive coverage drop. Are you okay with that?

Uh, I just see a coverage increase by 0,35%? Where do you see room for improvement? 😄

**Comment by Skarlso** [comment]
Uh. What? I swear this was decrees a moment ago... :D Okay, never mind. :D I'll review this stuff later today. :)

File: handlers/user.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func UserLogin(c echo.Context) error {
 	// Authenticate user
 	user, err := storeService.UserAuth(u, true)
 	if err != nil || user == nil {
-		gaia.Cfg.Logger.Error("invalid credentials provided", "message", err.Error())
+		gaia.Cfg.Logger.Info("invalid credentials provided", "username", u.Username)
```
Comment (Skarlso) [inline handlers/user.go]: How come this is no longer an error?

**Comment by Skarlso** [comment]
@michelvocks this was quiet the chunk to review man. :D

**Comment by michelvocks** [comment]
Yeah it's a lot but I think it's still ok for one PR, right?

Anyway, thanks a lot for review! 🤗 ❤️

**Pull request merged**